components/TextEditor/TextEditorContainer.module.scss

```diff
@@ -2,9 +2,12 @@
 .editorBox {
   border-style: solid;
   border-width: 5px;
+  width: available;
```

Comment:
Does this work? It doesn't seem to be listed as a valid value for `width`?

Reply:
Hmm... It is in Rider but not in Chrome so probably not doing anything. I'll remove